File: A's/py/116A.py
def main() -> None:
    n: int = int(input())
    
    passengers: int = 0
    cap: int = 0

    while n:
        a, b = map(int, input().split())

        passengers -= a
        passengers += b
        cap = max(cap, passengers)

        n -= 1

    print(cap)

if __name__ == "__main__":
    main()

File: A's/cpp/116A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n; cin >> n;

    int passengers = 0;
    int cap = 0;

    while (n--) {
        int a, b; cin >> a >> b;
        passengers -= a;
        passengers += b;
        cap = max(cap, passengers);
    }

    cout << cap << endl;
    
    return 0;
}